compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -150,7 +150,9 @@ object Completion {
       nameToSymbols.map { case (name, symbols) =>
         val typesFirst = symbols.sortWith((s1, s2) => s1.isType && !s2.isType)
         val desc = description(typesFirst)
-        Completion(name.toString, desc, typesFirst)
+        val strName = name.toString
+        val label = if (strName == "$u2192") "→" else strName // TODO fix name.decode and use it
+        Completion(label, desc, typesFirst)
```

Comment:
This *looks* unrelated and untested, and more fragile than the rest? If it's worth the technical debt, okay...

Reply:
I will open a separate issue for this when we merge it

Reply:
Removed this hack